client/reader/feed-stream/empty.jsx

```diff
@@ -21,15 +20,15 @@ var FeedEmptyContent = React.createClass( {
 
 	render: function() {
 		var action = (
-			<ExternalLink
+			<a
 				className="empty-content__action button is-primary"
 				onClick={ this.recordAction }
-				href="https://wordpress.com/fresh/">{ this.translate( 'Explore our Editors\' Picks' ) }</ExternalLink> ),
+				href="/discover">{ this.translate( 'Explore Discover' ) }</a> ),
```

Comment:
I think "Discover" works fine (succinct!).

Not sure about "Explore our Editors' Picks", as Discover is more than picks and includes features, sites (and soon collections).

Other options:

Discover great reads
Discover new sites
Discover
Explore the community
Explore WordPress